build-tools/automation/yaml-templates/use-dot-net.yaml

```diff
@@ -0,0 +1,27 @@
+# This template enables installing master/nighly builds of .NET 5.
+# We can eventually switch back to the UseDotNet task.
+
+parameters:
+  version: ''
```

Comment:
Minor nit would be to use `$(DotNetCoreVersion)` as the default parameter value here, but this looks good to me overall.